guide/payments/coin-selection.md

```diff
@@ -0,0 +1,312 @@
+
+There are two categories of coin selection strategies that are used in bitcoin applications:
+
+1. **Automatic Coin Selection** (wallet is delegated to control coin selection on behalf on user)
+2. **Manual Coin Selection** (user controls coin selection)
+
+{% include picture.html
+   image = "/assets/images/guide/glossary/coin-selection/funding-tx.jpg"
+   retina = "/assets/images/guide/glossary/coin-selection/funding-tx@2x.jpg"
+   mobile = "/assets/images/guide/glossary/coin-selection/funding-tx-mobile.jpg"
+   mobileRetina = "/assets/images/guide/glossary/coin-selection/funding-tx-mobile@2x.jpg"
+   alt-text = "Funding TX"
+   width = 800
+   height = 400
+%}
+
+### Hard contraints
+
+Before exploring some popular coin selection strategies, it’s worth noting the following when making a bitcoin payment in relation to UTXOs. 
+
+Firtstly; the transaction must have sufficient funding. For example, in order to send X, the transaction inputs must at least equate to X. Second; as blockspace is limited, miners prioritise transactions by *fee per byte* to maximise revenue. Transactions must be accompanied by the minimum relay fee for confirmation (therefore inputs must equal X + fee). Third; transaction outputs must have a value of 500 sats or more. Anything less than this value is considered  a "dust" output, and these transactions are often not relayed and confirmed by most nodes and miners in the network. Finally; it is essential for wallet owners to label their incoming and outgoing (change) coins. Labels provide context to a wallet’s transaction history and coin provenance, making it easier to successfully employ coin control for privacy optimisation.
+
+### Optimisations
+
+Coin selection will always lead to certain effects and outcomes which can either optimise or hinder our bitcoin payments. Below is a breakdown of the three most common effects of coin control.
+
+#### Cost
+
+Miners select transactions on the basis of their fee, measured by the fee rate of satoshis per byte. Therefore, it is cost effective for users to minimize transaction size by minimising the number of inputs.
+
+#### Speed
+
+Higher fees will increase the likelihood of your transaction being prioritised by miners and included in the next block, thereby increasing the speed of your transaction. This can be optimised by either increasing the fee rate, or by selecting a higher number of transaction inputs.
+
+#### Privacy
+
+Because each UTXO can be traced backwards on a public ledger, we can unearth the digital footprints of bitcoin payments. Privacy is often lost due to the exposure of unknown coins, addresses, balances, and economic activity to unrecognised trading partners. Privacy can be optimised by selecting transaction inputs from an anonymity set (CoinJoins or mixers) or a recognised label or cluster, or by reducing the size of change outputs. 
```

Comment:
```suggestion
Because each UTXO can be traced backward on a public ledger, we can unearth the digital footprints of Bitcoin payments. Privacy is often lost due to the exposure of unknown coins, addresses, balances, and economic activity to unrecognized trading partners. Privacy can be optimized by selecting transaction inputs from an anonymity set (CoinJoins or mixers) or a recognized label or cluster or reducing the size of change outputs. 
```